src/openforms/js/components/admin/form_design/logic/actions/Actions.js

```diff
@@ -225,6 +226,102 @@ const ActionSetRegistrationBackend = ({action, errors, onChange}) => {
   );
 };
 
+const ActionEvaluateDMN = ({action, errors, onChange}) => {
+  const intl = useIntl();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const convertMappingForward = mapping => {
+    if (!mapping) return [];
+
+    return Object.entries(mapping).map(([formVarKey, dmnVarKey]) => ({
+      formVar: formVarKey,
+      dmnVar: dmnVarKey,
+    }));
+  };
+
+  const convertMappingBackwards = mapping => {
+    if (!mapping) return {};
+
+    let convertedMapping = {};
+    mapping.map(item => {
+      convertedMapping[item.formVar] = item.dmnVar;
+    });
+    return convertedMapping;
```

Comment:
FYI - there also exists `Object.fromEntries` which is the reverse of `Object.entries` :)